modules/bot.py

```diff
@@ -58,7 +126,7 @@ def __init__(self, pickup, players):
         self.pickup = pickup
         self.channel = pickup.channel.channel
         self.winner = None
-        self.unpicked = []
```

Comment:
I removed all references to self.unpicked in favor of using unpicked pool instead. No tech debt introduced as a result!